lab1/Syntax.in.txt

```diff
@@ -0,0 +1,23 @@
+program = "begin" ";" decllist ";" cmpdstmt end ";"
+decllist = declaration | declaration ";" decllist
+declaration = "declare as " type ":" IDENTIFIER ";"
+type1 = "boolean" | "character" | "int" | “string”
+arraydecl = "list" "[" nr "]" "OF" type1
+type = type1|arraydecl
+cmpdstmt = "{" stmtlist "}"
+stmtlist = stmt | stmt ";" stmtlist
+stmt = simplstmt | structstmt
+simplstmt = assignstmt | iostmt
+assignstmt = IDENTIFIER "=" expression ";"
+expression = expression ("+" | "-") term | term
+term = term ("*" | "/") factor | factor
+factor = "(" expression ")" | IDENTIFIER | int | indexidentif | const
+indexidentif = IDENTIFIER "[" int "]"
+iostmt = "read" | "write" "(" IDENTIFIER ")" ";"
+structstmt = cmpdstmt | ifstmt | whilestmt | forstmt
+ifstmt = "verify" "(" condition ")" cmpdstmt ["elverify" "(" condition ")" cmpdstmt] ["else" cmpdstmt]
```

Comment:
be careful: here you can only have one "elverify" statement (consider changing from [] to {} brackets)